mava/components/updating/parameter_server.py

```diff
@@ -196,6 +211,13 @@ def on_parameter_server_add_to_parameters(
         """
         # server.store._add_to_params set by Parameter Server
         params: Dict[str, Any] = server.store._add_to_params
+
+        # Interrupt is a flag that sent info about whether the executor is failing
+        if "interrupt" in params.keys():
```

Comment:
Can we give this flag a more descriptive name? Maybe something like `"executor_failed"`?

Reply:
Yeah sure!

Reply:
I removed this if statement because I found out that add method help in incrementing the values